src/renderer/api/catalog-entity-registry.ts

```diff
@@ -65,7 +65,7 @@ export class CatalogEntityRegistry {
 
   getItemsForCategory<T extends CatalogEntity>(category: CatalogCategory): T[] {
     const supportedVersions = category.spec.versions.map((v) => `${category.spec.group}/${v.name}`);
-    const items = this._items.filter((item) => supportedVersions.includes(item.apiVersion) && item.kind === category.spec.names.kind);
+    const items = this._items.filter((item) => supportedVersions.includes(item?.apiVersion) && item.kind === category.spec.names.kind);
```

Comment:
I think it would be better if `this._items` filtered out null or undefined values.

Reply:
thanks, good point

Reply:
Fixed in https://github.com/lensapp/lens/pull/2814/commits/b0cff92ca804ee91bf8aaa784ab2571fba2a7967

Reply:
I meant on line 41. Namely keep the "database" clean.

Reply:
ah, ok, sorry, fixed https://github.com/lensapp/lens/pull/2814/commits/0da59e998ed2fd3cc38251f6a6f5155c571ca050